change/@fluentui-react-charting-4ff1470e-7359-4539-ac6d-724412f4cd67.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30469&commitId=e67436f5849bfca58b9aadebe01be9290ff48745&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-DonutChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-DonutChart.Basic - Dark Mode.default.chromium.png|809| Changed|
|react-charting-DonutChart.Dynamic - Dark Mode.default.chromium.png|634| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Events - Dark Mode.default.chromium.png|8| Changed|
|react-charting-LineChart.Events.default.chromium.png|54| Changed|
|react-charting-LineChart.Events - RTL.default.chromium.png|17| Changed|

</details>


  <details><summary><b>react-charting-SparkLineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SparkLineChart.Basic - Dark Mode.default.chromium.png|12| Changed|
|react-charting-SparkLineChart.Basic.default.chromium.png|71| Changed|
|react-charting-SparkLineChart.Basic - RTL.default.chromium.png|71| Changed|

</details>


  <details><summary><b>react-charting-TreeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Basic.default.chromium.png|1079| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|183713| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|1079| Changed|
|react-charting-TreeChart.Compact.default.chromium.png|1070| Changed|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|181595| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|1070| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
outdated screenshots.